src/Libraries/RevitNodesUI/RevitDropDown.cs

```diff
@@ -478,7 +478,10 @@ protected override SelectionState PopulateItemsCore(string currentSelection)
                 if (category != null)
                 {
                     string name = getFullName(category);
-                    Items.Add(new DynamoDropDownItem(name, categoryId));
+                    if (name != "Cameras")
```

Comment:
hmm... I don't think this will work in all versions of Revit... @sm6srw what do you think?